ipatests/prci_definitions/nightly_master.yaml

```diff
@@ -628,14 +628,137 @@ jobs:
         timeout: 7200
         topology: *master_1repl_1client
 
-  fedora-28/test_webui:
+  fedora-28/test_webui_cert:
     requires: [fedora-28/build]
     priority: 50
     job:
       class: RunWebuiTests
       args:
         build_url: '{fedora-28/build_url}'
-        test_suite: test_webui/
+        test_suite: test_webui/test_cert.py
         template: *ci-master-f28
-        timeout: 19600
+        timeout: 1800
```

Comment:
I guess we can adjust/verify timeouts by looking into the last NR run of the test_webui execution logs.

Reply:
test_cert ~ 400s
other tests ~ 1000-2000s
So we can try with these timeouts (1800 and 3600). But for some tests they can be increased if necessary.